homeassistant/components/halohome/light.py

```diff
@@ -0,0 +1,61 @@
+"""HALO Home integration light platform."""
+import halohome
+
+from homeassistant.components.light import (
+    ATTR_BRIGHTNESS,
+    ATTR_COLOR_TEMP,
+    COLOR_MODE_COLOR_TEMP,
+    LightEntity,
+)
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.core import HomeAssistant
+
+from .const import DOMAIN
+
+
+async def async_setup_entry(
+    hass: HomeAssistant, entry: ConfigEntry, async_add_entities
+):
+    """Set up the HALO Home light platform from a config entry."""
+    connection = hass.data[DOMAIN][entry.entry_id]
+    async_add_entities(HaloLight(device) for device in connection.devices)
+
+
+class HaloLight(LightEntity):
```

Comment:
There is no update here, should this be marked as `assumed_state` instead?

See: <https://developers.home-assistant.io/docs/core/entity/#generic-properties>